source/admin_manual/list_of_events.rst

```diff
@@ -2378,561 +1519,525 @@ obox plugin
+
+
+.. dovecot_core:event:: metacache_mailbox_bundle_download_finished
+   :added: v2.3.11
+   :inherit: mailbox, metacache_bundle_download
+
+   Mailbox index bundle file was downloaded (can happen while the mailbox is
+   being refreshed).
+
+
+.. dovecot_event:field_group:: metacache_upload
+
+   :field error: Error message if the upload failed.
+
+
+.. dovecot_core:event:: metacache_upload_started
+   :added: v2.3.11
+   :inherit: metacache_upload
+
+   Changes in metacache are being uploaded to storage.
+
+   .. todo:: Does this inherit from mail_user or mailbox?
+
+
+.. dovecot_core:event:: metacache_upload_finished
+   :added: v2.3.11
+   :inherit: metacache_upload
+
+   Changes in metacache were uploaded to storage.
+
+   .. todo:: Does this inherit from mail_user or mailbox?
+
+
+.. dovecot_event:field_group:: metacache_bundle_upload
+
+   :field filename: Bundle filename.
+   :field bundle_type: Bundle type: ``diff``, ``base``, or ``self``.
+   :field bundle_size: Size of the bundle file in bytes (uncompressed).
+   :field mailbox_guid: GUID of the mailbox being uploaded. Note that the
+     mailbox name field may or may not exist in this event depending on
+     whether a single mailbox or the whole user is being uploaded.
+   :field reason: Reason for what changed in the indexes to cause this bundle
+     to be created and uploaded.
+   :field error: Error message if the upload failed.
+
+
+.. dovecot_core:event:: metacache_user_bundle_upload_started
+   :added: v2.3.11
+   :inherit: mail_user, mailbox, metacache_bundle_upload
+
+   User index bundle file is being uploaded. Can happen while the user is
+   being uploaded.
+
+   .. note:: This event can be inherited from a mailbox event, and include the
+             mailbox fields, if the user upload was triggered by a mailbox
+             upload.
+
+
+.. dovecot_core:event:: metacache_user_bundle_upload_finished
+   :added: v2.3.11
+   :inherit: metacache_bundle_upload
+
+   User index bundle file was uploaded. Can happen while the user is being
+   uploaded.
+
+   .. note:: This event can be inherited from a mailbox event, and include the
+             mailbox fields, if the user upload was triggered by a mailbox
+             upload.
+
+
+.. dovecot_core:event:: metacache_mailbox_bundle_upload_started
+   :added: v2.3.11
+   :inherit: metacache_bundle_upload
+
+   Mailbox index bundle file is being uploaded. Can happen while the mailbox
+   is being uploaded.
+
+
+.. dovecot_core:event:: metacache_mailbox_bundle_upload_finished
+   :added: v2.3.11
+   :inherit: metacache_bundle_upload
+
+   Mailbox index bundle file was uploaded. Can happen while the mailbox is
+   being uploaded.
+
+
+.. dovecot_event:field_group:: metacache_clean
+
+   :field min_priority: Which priority indexes are being cleaned.
+   :field error: Error message if the upload failed.
+
+
+.. dovecot_core:event:: metacache_user_clean_started
+   :added: v2.3.11
+   :inherit: mail_user, metacache_clean
+
+   TODO
```

Comment:
User is started to be cleaned from metacache, either entirely or only partially (only low priority files).